Assets/HoloToolkit-Examples/Input/Scenes/InteractiveMeshCursor.unity

```diff
@@ -362,6 +363,94 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 906323c940a3fad4f8f7e9e4fcd747f4, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  containerObject: {fileID: 0}
+  alignmentType: 0
+  stationarySpaceTypePosition: {x: 0, y: 0, z: 0}
+  roomScaleSpaceTypePosition: {x: 0, y: 0, z: 0}
+--- !u!1 &1166889112
+GameObject:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  serializedVersion: 5
+  m_Component:
+  - component: {fileID: 1166889117}
+  - component: {fileID: 1166889116}
+  - component: {fileID: 1166889115}
+  - component: {fileID: 1166889114}
+  m_Layer: 0
+  m_Name: CubeToStandOn
```

Comment:
Is this adding a new cube to test out the new setting for teleporting? I don't think `InteractiveMeshCursor` is the right scene for that. Maybe `MotionControllerTest` or a new `TeleportingTest` scene?

Reply:
It is adding an extra cube indeed. Do you see that as a real problem? There is a cube in the scene now, it's just a bit too high to easily place the cursor on. If you want it gone, I will delete it again

Reply:
The only problem I see is discoverability, if this new cube is intended to be the test for this new feature, since it's in a cursor test scene.